docs/wiki/release/Betaflight-4-5-Release-Notes.md

```diff
@@ -39,11 +39,12 @@ Always test new firmware carefully and in a controlled environment!
 - [Improved landings for wings](#15-keep-i-term-at-zero-for-fixed-wings-at-zero-throttle) iTerm is now kept active for wings while gliding into land, at zero throttle.
 - [Changed Soft Serial pin assignment command](#16-changed-soft-serial-pin-assignment-cli-command) Use `RESOURCE SOFTSERIALTX 1 <pin>` instead of `RESOURCE SERIALTX11 <pin>`.
 - [Custom Build Options](#16-custom-build-options):
-    - [RPM Limiter](#161-rpm-limiter-build-option) Limits the maximum average RPM, for Spec racing.
-    - [Quick OSD](#162-quick-osd-menu-build-option) Adds an OSD page that allows the user to make most race-related changes in one place.
-    - [RC Stats](#163-rc-stats-osd-build-option) Modified Stats screen including throttle summary data. 
-    - [Pre-Arm page](#164-pre-arm-spec-race-settings-osd-build-option) Pre-arm screen that displays values of relevance to spec racing
-    - [GPS Lap Timer](#165-gps-lap-timer) Ever wanted to fly laps at a park and time yourself, without complicated extra hardware?  Add a GPS module and you can do exactly that.
+	- [RACE_PRO](#161-race-pro-build-option) Includes RPM Limiter, Quick OSD, RC Stats, Pre-arm page, and changes a few default settings.
```

Comment:
4 spaces, not tab-character. please modify your editor.

Reply:
Oh dam sorry about it